src/assets/styles/app.scss

```diff
@@ -0,0 +1,10 @@
+// ----------------------------------
+// UI
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
```

Comment:
Could we insert `@import './src/assets/styles/base';` here instead of inside each of the component files or would that break the structure?

Reply:
I did think about this (albeit briefly), but it's tough to come to a decision. Factors at play:

- what happens if/when the rest of the Sass folders start emitting styles?
- does `@import`ing `_base.scss` in to `app.scss`, along with the component styles confuse the ordering/structure of styles?
- do the above cases create an argument for breaking down `styles/` in to two folders - one which **does** emit CSS, and one which **doesn't** (ie. `styles/base` & `styles/utils`)

I feel like, for now, the current mode makes sense as we're not emitting any CSS from the `styles/` folders and `@import`ing `_base.scss` in to individual components makes it explicitly clear that the component relies on the base theme settings.

Reply:
Lots of maybes and things to consider so probably worth us having a more in-depth chat about this to go through the options

Reply:
Happy with that - can include indents from https://github.com/wellcometrust/corporate-components/pull/19#discussion_r349021872 in that chat, too? Though more of a simpler issue of preference so should be more straight-forwards to resolve

Reply:
To do: 

- investigate this compat. with stylelint